bodhi/server/models.py

```diff
@@ -476,12 +476,28 @@ class TestCase(Base):
 
 
 class Package(Base):
+    """
+    This model represents a package.
+
+    This model uses single-table inheritance to allow for different package types.
+
+    Attributes:
+        name (unicode): A unicode string that uniquely identifies the package.
+        requirements (unicode): A unicode string that describes the requirements of a package.
+        type (int): The polymorphic identity column. This is used to identify what Python
+            class to create when loading rows from the database.
+        builds (list): A list of :class:`Build` objects.
+        test_cases (list): A list of :class:`TestCase` objects.
+        committers (list): A list of :class:`User` objects who are committers.
```

Comment:
I thought they were just lists because I checked on Anitya, but it turned out that was a property making them a list. It's actually an SQLAlchemy type. I updated the docs.